src/lib/get-verified-fetch.ts

```diff
@@ -95,7 +95,8 @@ export async function libp2pDefaults (config: Libp2pDefaultsOptions): Promise<Li
 
   if (config.enableWss) {
     transports.push(webSockets())
-    filterAddrs.push('wss')
+    filterAddrs.push('wss') // /dns4/sv15.bootstrap.libp2p.io/tcp/443/wss/p2p/QmNnooDu7bfjPFoTZYxMNLWUQJyrVwtbZg5gBMjTezGAJ
+    filterAddrs.push('ws') // /ip4/A.B.C.D/tcp/4002/tls/sni/A-B-C-D.peerid.libp2p.direct/ws/p2p/peerid
```

Comment:
```suggestion
    filterAddrs.push('tls') // /ip4/A.B.C.D/tcp/4002/tls/sni/A-B-C-D.peerid.libp2p.direct/ws/p2p/peerid
```

Reply:
I guess this works for this project (we get both `/ws` and `/http` that work in secure context)

Reply:
But wait, this way we co-mingle `ws` and `http`

Reply:
Pushed https://github.com/ipfs/service-worker-gateway/pull/405/commits/36d006638297836a32d6e1f4d95e1b9030abaf7a to ensure we also include `/tls` for HTTP ones.

Reply:
Sounds good.